ansible/roles/debops.prosody/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+
+
+- name: Install required packages
+  package:
+    name: '{{ item }}'
+    state: 'present'
+  with_flattened:
+    - '{{ prosody__base_packages }}'
+
```

Comment:
You could add here an additional `prosody__packages` variable to allow installation of additional APT packages with Prosody, if needed.